README.md

```diff
@@ -1,9 +1,19 @@
 # Stedi EDI Bootstrap
 
-This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use cases. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use cases. Your solution may differ, depending on your systems and requirements. 
 
+## Hands-On Support
 
-## Read and Write Workflow 
+We'd like to set up and customize the bootstrap repository with you. Working together helps us understand what Stedi customers need and helps get your solution into production as quickly as possible. We offer free hands-on support that includes:
+
+- Help deploying the bootstrap workflows and customizing them for your use cases
+- Best practices for designing a scalable Stedi integration
+- EDI experts to answer your questions
+- Live troubleshooting over slack or video call
```

Comment:
```suggestion
- Live troubleshooting over Slack or video call
```

Reply:
I’ll resist putting period after each list item, but Slack really must be capitalized. 😉